locals.tf

```diff
@@ -1,4 +1,45 @@
 locals {
+  # Manage certificates
+  pre_install_certificate_1 = (
```

Comment:
issue: I think something like `pre_install_gitlab_certificate` makes it clearer.